indico/web/client/js/react/util/Responsive.jsx

```diff
@@ -60,3 +60,18 @@ export default Object.assign(Responsive, {
   Tablet: factory(DIMENSIONS.tablet, DIMENSIONS.computer),
   Phone: factory(null, DIMENSIONS.tablet),
 });
+
+export function useResponsive() {
+  return {
+    isWideScreen: useMediaQuery({query: `(min-width: ${DIMENSIONS.computer}px)`}),
+    isDesktop: useMediaQuery({
+      query: `(min-width: ${DIMENSIONS.computer}px) and (max-width: ${DIMENSIONS.wideScreen -
+        1}px)`,
```

Comment:
this wrapping here is super awful :/

Reply:
defining

    const belowWideScreen = DIMENSIONS.wideScreen - 1;

and then using this here would avoid the wrapping